Justfile

```diff
@@ -97,15 +97,28 @@ codecov RULE:
     export RUSTC_WORKSPACE_WRAPPER="{{ absolute_path("scripts/coverage-wrapper-rustc") }}"
     {{ just_executable() }} {{ RULE }}
     mkdir -p coverage/codecov
-    cargo llvm-cov report --profile dev-release --codecov --output-path coverage/codecov/new.json
+    # If coverage profiling produced corrupted/empty .profraw (e.g. due to a segfault),
+    # `cargo llvm-cov report` can fail sometimes. Retry once before falling back to an empty JSON.
+    for attempt in 1 2; do
+        cargo llvm-cov report --profile dev-release --codecov --output-path coverage/codecov/new.json && break
+        if [ $attempt -eq 1 ]; then
+            echo "llvm-cov report failed, retrying..."
+            sleep 1
```

Comment:
Sleep is not necessary.

Reply:
On second thought you are right, it's easy to rerun the CI in the rare case someone needs the coverage and it fails